### PR TITLE
Add Rule0068 to LinterCop.ruleset.json

### PR DESCRIPTION
**Pull request by StefanMaron**
This rule checks if the current object has missing permission when data access of any kind is used.

### CODE REVIEW

File: BusinessCentral.LinterCop/Design/Rule0068CheckObjectPermission.cs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+            if (buildInTableDataReadMethodNames.Contains(operation.TargetMethod.Name.ToLowerInvariant()))
+                CheckProcedureInvocation(objectPermissions, variableType, 'r',  ctx);
+            else if (buildInTableDataInsertMethodNames.Contains(operation.TargetMethod.Name.ToLowerInvariant()))
+                CheckProcedureInvocation(objectPermissions, variableType, 'i', ctx);
+            else if (buildInTableDataModifyMethodNames.Contains(operation.TargetMethod.Name.ToLowerInvariant()))
+                CheckProcedureInvocation(objectPermissions, variableType, 'm', ctx);
+            else if (buildInTableDataDeleteMethodNames.Contains(operation.TargetMethod.Name.ToLowerInvariant()))
+                CheckProcedureInvocation(objectPermissions, variableType, 'd', ctx);
+
```
Comment (guidorobben) [inline BusinessCentral.LinterCop/Design/Rule0068CheckObjectPermission.cs:82]: empty line

**Comment by StefanMaron** [comment]
added the wiki page
https://github.com/StefanMaron/BusinessCentral.LinterCop/wiki/LC0068

**Comment by StefanMaron** [comment]
@Arthurvdv 
Please also have a look at the test step in the pipelines I added, if needed I can move that to another branch of course.
But it seems like it does not work for all compiler versions and I am not sure what the reason is for this.

**Comment by StefanMaron** [comment]
moved tests to its own branch
https://github.com/StefanMaron/BusinessCentral.LinterCop/pull/723

**Pull request merged**